src/components/modals/TransactionAccelerateTransactionModal.js

```diff
@@ -0,0 +1,243 @@
+// @flow
+import React, { PureComponent } from 'react'
+import { connect } from 'react-redux'
+import { ActivityIndicator, Alert, Text, View } from 'react-native'
+import { Actions } from 'react-native-router-flux'
+import type { EdgeCurrencyWallet, EdgeDenomination, EdgeTransaction, EdgeSpendInfo } from 'edge-core-js'
+
+import { sprintf } from 'sprintf-js'
+
+import { playSendSound } from '../../actions/SoundActions.js'
+import { TRANSACTION_DETAILS } from '../../constants/indexConstants'
+import s from '../../locales/strings.js'
+import { Slider } from '../../modules/UI/components/Slider/Slider.ui.js'
+import { getSelectedWallet } from '../../modules/UI/selectors.js'
+import { type RootState } from '../../types/reduxTypes.js'
+import type { GuiWallet } from '../../types/types.js'
+import * as UTILS from '../../util/utils.js'
+import { type Theme, type ThemeProps, cacheStyles, withTheme } from '../services/ThemeContext.js'
+import { ModalTitle, ModalMessage, ModalCloseArrow } from '../themed/ModalParts.js'
+import { ThemedModal } from '../themed/ThemedModal.js'
+import { Tile } from '../themed/Tile.js'
+import { type AirshipBridge } from './modalParts'
+
+type Props = OwnProps & StateProps & ThemeProps
+type OwnProps = {
+  bridge: AirshipBridge<Status>,
+  edgeTransaction: EdgeTransaction,
+  guiWallet: GuiWallet,
+  walletDefaultDenomProps: EdgeDenomination
+}
+type StateProps = {
+  wallet: EdgeCurrencyWallet
+}
+
+type State = {
+  edgeUnsignedTransaction: ?EdgeTransaction,
+  error: ?Error,
+  status: Status
+}
+
+type Status = 'confirming' | 'sending' | 'sent'
+
+class TransactionAccelerateModalComponent extends PureComponent<Props, State> {
+  constructor() {
+    super()
+
+    this.state = {
+      edgeUnsignedTransaction: null,
+      error: null,
+      status: 'confirming'
+    }
+  }
+
+  componentDidMount() {
+    this.makeRbfTransaction()
+  }
+
+  componentWillUnmount() {
+    this.resetRbfState()
+  }
+
+  makeRbfTransaction = async () => {
+    const { edgeTransaction, wallet } = this.props
+
+    if (edgeTransaction.spendTargets) {
+      const spendTargets = edgeTransaction.spendTargets.map(spendTarget => ({
+        nativeAmount: spendTarget.nativeAmount,
+        publicAddress: spendTarget.publicAddress,
+        uniqueIdentifier: spendTarget.uniqueIdentifier
+      }))
+      const rbfTxid = edgeTransaction.txid
+
+      const edgeSpendInfo: EdgeSpendInfo = {
+        spendTargets,
+        rbfTxid
+      }
```

Comment:
Add currency code from `edgeTransaction.spendTargets[0]` to the `edgeSpendInfo`.